src/components/backend-ai-resource-policy-list.ts

```diff
@@ -209,88 +243,92 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
             <fieldset>
               <paper-input type="text" name="new_policy_name" id="id_new_policy_name" label="Policy Name"
                            required
-                           error-message="Policy name only accepts letters and numbers"></paper-input>
+                           error-message="Policy name only accepts letters and numbers" style="width:100%;"></paper-input>
               <h4>Resource Policy</h4>
               <div class="horizontal center layout">
-                <paper-dropdown-menu id="cpu-resource" label="CPU">
-                  <paper-listbox slot="dropdown-content" selected="0">
-                  ${this.cpu_metric.map(item => html`
-                    <paper-item value="${item}">${item}</paper-item>
-                  `)}
-                  </paper-listbox>
-                </paper-dropdown-menu>
-                <paper-dropdown-menu id="ram-resource" label="RAM (GB)">
-                  <paper-listbox slot="dropdown-content" selected="0">
-                  ${this.ram_metric.map(item => html`
-                    <paper-item value="${item}">${item}</paper-item>
-                  `)}
-                  </paper-listbox>
-                </paper-dropdown-menu>
-                <paper-dropdown-menu id="gpu-resource" label="GPU">
-                  <paper-listbox slot="dropdown-content" selected="0">
-                  ${this.gpu_metric.map(item => html`
-                    <paper-item value="${item}">${item}</paper-item>
-                  `)}
-                  </paper-listbox>
-                </paper-dropdown-menu>
-                <paper-dropdown-menu id="fgpu-resource" label="fgpu">
-                  <paper-listbox slot="dropdown-content" selected="0">
-                  ${this.fgpu_metric.map(item => html`
-                    <paper-item value="${item}">${item}</paper-item>
-                  `)}
-                  </paper-listbox>
-                </paper-dropdown-menu>
+                  <div class="vertical layout" style="width:75px; margin: 0px 10px 0px 0px;">
+                    <wl-label>CPU</wl-label>
+                    <wl-textfield id="cpu-resource" type="number"
+                    @change="${(e) =>this._validateResourceInput(e)}"
```

Comment:
This might be a personal preference, but why don't you add two tabs (four spaces) before `@change` attribute for readibility of unclosed tag?

Reply:
Also, there should be a space after `=>` operator.